README.md

```diff
@@ -4,14 +4,16 @@ This is a [composite GitHub Action](https://docs.github.com/en/actions/creating-
 
 This actions rolls into a single step two tasks that are commonly run together:
 
-1. Build a Python package's distribution, accomplished with the PyPA's [build](https://pypa-build.readthedocs.io/en/stable/index.html) tool.
-2. Upload the distribution to PyPI, using the [pypa/gh-action-pypi-publish](https://github.com/pypa/gh-action-pypi-publish) action, using PyPI's [trusted publishers](https://docs.pypi.org/trusted-publishers/) mechanism. **New in v2: using trusted publishers is required. Use v1 for the legacy account token authentication method.**
+1. Build a Python package's distribution
+2. Upload the distribution to PyPI, using PyPI's [trusted publishers](https://docs.pypi.org/trusted-publishers/) mechanism. **New in v2: using trusted publishers is required. Use v1 for the legacy account token authentication method.**
+
+Both actions are done with [uv](https://docs.astral.sh/uv/).
 
 > **Note**
```

Comment:
```suggestion
> [!NOTE]
```

We can use the new-ish [GitHub Markdown alerts syntax now](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).